Src/Newtonsoft.Json/Linq/JArray.cs

```diff
@@ -302,25 +302,25 @@ public int IndexOf(JToken item)
         }
 
         /// <summary>
-        /// Inserts an item to the <see cref="T:System.Collections.Generic.IList`1"/> at the specified index.
+        /// Inserts an item to the <see cref="IList{T}"/> at the specified index.
```

Comment:
Should this be `<see cref="JArray"/>`? The original comments are inherited from `IList<T>`.

Can be repeated for other `IList<T>` crefs around here.